spec/features/issues_spec.rb

```diff
@@ -323,112 +323,6 @@
           include_examples 'deleted item is listed in Trash', :issue
           include_examples 'recover deleted item', :issue
         end
-
-        describe 'add evidence', js: true do
```

Comment:
We need an equivalent feature spec that confirms you can add Evidence from Issues#show.

There's quite a bit here actually: filtering of nodes, assignment of parent node, assignment of author, etc. We cannot just scrap all this desired behaviour from the test suite.